src/app/system/configurations/configurations.component.ts

```diff
@@ -0,0 +1,78 @@
+import {Component, OnInit, ViewChild} from '@angular/core';
+import {MatPaginator, MatSort, MatTableDataSource} from '@angular/material';
+import {ActivatedRoute, Router} from '@angular/router';
+
+/** Custom Services */
+import {SystemService} from '../system.service';
+
+
+@Component({
+  selector: 'mifosx-configurations',
```

Comment:
```suggestion
  selector: 'mifosx-global-configurations',
```
Update and make all necessary changes to file names as well.

Reply:
Should I change the `edit-configuration` to `edit-global-configuration` too?